static/app/views/performance/browser/webVitals/components/webVitalMeters.tsx

```diff
@@ -25,64 +25,65 @@ export default function WebVitalMeters({onClick, projectData, projectScore}: Pro
   };
   return (
     <Container>
-      <Flex>
-        <MeterBarContainer key="lcp" onClick={() => onClick?.('lcp')}>
-          <MeterBarBody>
-            <MeterHeader>{t('Largest Contentful Paint (P75)')}</MeterHeader>
-            <MeterValueText>
-              {getFormattedDuration(
-                (projectData?.data?.[0]?.['p75(measurements.lcp)'] as number) / 1000
-              )}
-            </MeterValueText>
-          </MeterBarBody>
-          <MeterBarFooter score={projectScore.lcpScore} />
-        </MeterBarContainer>
-        <MeterBarContainer key="fcp" onClick={() => onClick?.('fcp')}>
-          <MeterBarBody>
-            <MeterHeader>{t('First Contentful Paint (P75)')}</MeterHeader>
-            <MeterValueText>
-              {getFormattedDuration(
-                (projectData?.data?.[0]?.['p75(measurements.fcp)'] as number) / 1000
-              )}
-            </MeterValueText>
-          </MeterBarBody>
-          <MeterBarFooter score={projectScore.fcpScore} />
-        </MeterBarContainer>
-        <MeterBarContainer key="fid" onClick={() => onClick?.('fid')}>
-          <MeterBarBody>
-            <MeterHeader>{t('First Input Delay (P75)')}</MeterHeader>
-            <MeterValueText>
-              {getFormattedDuration(
-                (projectData?.data?.[0]?.['p75(measurements.fid)'] as number) / 1000
-              )}
-            </MeterValueText>
-          </MeterBarBody>
-          <MeterBarFooter score={projectScore.fidScore} />
-        </MeterBarContainer>
-        <MeterBarContainer key="cls" onClick={() => onClick?.('cls')}>
-          <MeterBarBody>
-            <MeterHeader>{t('Cumulative Layout Shift (P75)')}</MeterHeader>
-            <MeterValueText>
-              {formatAbbreviatedNumber(
-                projectData?.data?.[0]?.['p75(measurements.cls)'] as number,
-                2
-              )}
-            </MeterValueText>
-          </MeterBarBody>
-          <MeterBarFooter score={projectScore.clsScore} />
-        </MeterBarContainer>
-        <MeterBarContainer key="ttfb" onClick={() => onClick?.('ttfb')}>
-          <MeterBarBody>
-            <MeterHeader>{t('Time To First Byte (P75)')}</MeterHeader>
-            <MeterValueText>
-              {getFormattedDuration(
-                (projectData?.data?.[0]?.['p75(measurements.ttfb)'] as number) / 1000
-              )}
-            </MeterValueText>
-          </MeterBarBody>
-          <MeterBarFooter score={projectScore.ttfbScore} />
-        </MeterBarContainer>
-      </Flex>
+      {projectScore && (
```

Comment:
l: Can we early return if projectScore is undefined instead of the nested conditional? This also means we can easily add a undefined state if we need to in the future.

Ideally we can avoid ever passing in an `undefined` `projectScore` in the first place.